native/mjml_nif/src/lib.rs

```diff
@@ -19,9 +23,20 @@ pub struct RenderOptions <'a> {
     pub fonts: Option<HashMap<Term<'a>, Term<'a>>>
 }
 
+#[derive(NifStruct)]
+#[module = "Mjml.ParserOptions"]
+pub struct ParserOptions <'a> {
+    pub include_loader: Term<'a>,
+    pub local_loader_path: Option<String>
+}
+
 #[rustler::nif]
-pub fn to_html<'a>(env: Env<'a>, mjml: String, render_options: RenderOptions) -> NifResult<Term<'a>> {
-    return match mrml::parse(&mjml) {
+pub fn to_html<'a>(env: Env<'a>, mjml: String, render_options: RenderOptions, parser_options: ParserOptions) -> NifResult<Term<'a>> {
+    let parse_opts = parser::ParserOptions {
+        include_loader: include_loader_option(&parser_options),
+        ..parser::ParserOptions::default()
```

Comment:
Currently there is only one option in Rust's side, however, if they extend/mutate it, we won't need to change anything on our side (Elixir NIF). That's the reason.